app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.js

```diff
@@ -62,17 +119,105 @@ function setConfigureError(err) {
   });
 }
 
+export function enableGroupTrigger() {
+  const namespace = NamespaceStore.getState().selectedNamespace;
+  const pipelineTriggers = PipelineTriggersStore.getState().triggers;
+  const configMappings = pipelineTriggers.triggersGroupRunArgsToAdd;
+  let runArgsMapping = {
+    arguments: [],
+    pluginProperties: [],
+  };
+  let propertiesConfigToEnable = {};
+  configMappings.arguments.map((arg) => runArgsMapping.arguments.push(arg.arg));
+  configMappings.pluginProperties.map((plugin) =>
+    runArgsMapping.pluginProperties.push(plugin.plugin)
+  );
+  configMappings.propertiesConfig.map(
+    (property) =>
+      (propertiesConfigToEnable = { ...propertiesConfigToEnable, ...property.propertiesConfig })
+  );
+  const triggerName = pipelineTriggers.triggersGroupToAdd.map(
+    (trigger) => trigger.programId.application
+  );
+  const scheduleName = `${pipelineTriggers.pipelineName}.${namespace}.${triggerName.toString()}`;
+
+  const requestObj = {
+    name: scheduleName,
+    description: '',
+    program: {
+      programName: pipelineTriggers.workflowName,
+      programType: 'WORKFLOW',
+    },
+    properties: Object.assign(
```

Comment:
not sure what this object assign is doing?

Reply:
nice catch, no need to assign again